extensions/amp-carousel/amp-carousel-analytics.md

```diff
@@ -0,0 +1,78 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="amp-carousel-analytics"></a>AMP Carousel and Analytics
+
+## Carousel (slides) triggers
+
+`<amp-carousel type="slides">` issues events for major states . These events can be reported via an analytics package using triggers.
```

Comment:
yes. The configuration is where you would use these triggers. If this was copied from access, do you mind changing it there as well?